### PR TITLE
You can now tip borgs over

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Uses some code from https://github.com/Skyrat-SS13/Skyrat-tg/pull/10667 (specifically, removing hats from borgs when you flip them over)

This might not be that much of a popular opinion, so I have here hired a professional artist to draw a couple of comics to show off the capabilities this PR brings to the players:

backup

medicalcomic

engiecomic

clowncomic

defaultcomic

## Why It's Good For The Game

I hate Silicon

Teaches Silicon that they should still live within the Crew rather than above them, otherwise people will tip you over and no one would want to help tip you back up.

The timers also, for the most part, match handcuff timers, so there's already a human counterpart of this. Even if it wasn't intentional.

## Changelog

:cl: JohnFulpWillard and theOOZ
qol: Tipping mobs over all runs on right click, rather than running on both right and left click.
balance: You can now tip Borgs over.
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
>		self_right_time = 60 SECONDS)

No.

**Comment by HugoOdaX** [comment]
> > ```
> >   self_right_time = 60 SECONDS)
> > ```
> 
> No.

Yes.

File: code/datums/components/tippable.dm [hunk 1]
```diff
@@ -18,12 +18,13 @@
...
-		datum/callback/pre_tipped_callback,
-		datum/callback/post_tipped_callback,
-		datum/callback/post_untipped_callback)
+	tip_time = 3 SECONDS,
+	untip_time = 1 SECONDS,
+	self_right_time = 60 SECONDS,
+	datum/callback/pre_tipped_callback,
+	datum/callback/post_tipped_callback,
+	datum/callback/post_untipped_callback,
+)
```
Comment (LemonInTheDark) [inline code/datums/components/tippable.dm:27]: Why enforce a pattern you don't follow?

Comment (JohnFulpWillard) [reply code/datums/components/tippable.dm:27]: i wanted it to be consistent with other elements/components

**Comment by JohnFulpWillard** [comment]
I have finished my new comic, if I do make another one, it will likely be either engineering or bar related

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -4,6 +4,10 @@
 	spark_system.attach(src)
 
 	ADD_TRAIT(src, TRAIT_CAN_STRIP, INNATE_TRAIT)
+	AddComponent(/datum/component/tippable, \
+		tip_time = 2.5 SECONDS, \
```
Comment (Mothblocks) [inline code/modules/mob/living/silicon/robot/robot.dm:8]: Should probably be a fair bit longer, TBH, as funny as it is

**Comment by Sheits** [comment]
image

**Comment by timothymtorres** [comment]
What happens if you tip a borg while someone is riding it?

**Comment by Blevruz** [comment]
good pr + need borgs to get a "flip the bird" module

**Comment by JohnFulpWillard** [comment]
> What happens if you tip a borg while someone is riding it?

image

**Comment by theOOZ** [comment]
this is [plagiarism](https://github.com/Skyrat-SS13/Skyrat-tg/pull/10667)!!

**Comment by JohnFulpWillard** [comment]
> this is [plagiarism](https://github.com/Skyrat-SS13/Skyrat-tg/pull/10667)!!

actually wasnt aware of that but it actually does bring up the hat problem I wasn't aware of

**Comment by theOOZ** [comment]
> > this is [plagiarism](https://github.com/Skyrat-SS13/Skyrat-tg/pull/10667)!!
> 
> actually wasnt aware of that but it actually does bring up the hat problem I wasn't aware of

lets swap notes... i didn't consider buckling

**Comment by timothymtorres** [comment]
> > What happens if you tip a borg while someone is riding it?
> 
> image

This image is cursed.  You should probably prevent tipping over if someone is buckled to the borg.

**Comment by Sheits** [comment]
> > > What happens if you tip a borg while someone is riding it?
> > 
> > 
> > image
> 
> This image is cursed. You should probably prevent tipping over if someone is buckled to the borg.

borgs intentionally buckling people to prevent flipping

File: code/modules/mob/living/silicon/robot/robot.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/modules/mob/living/silicon/robot/robot.dm]: yeah but then you have to flash them to tip them

**Comment by SoreYew** [comment]
> > > What happens if you tip a borg while someone is riding it?
> 
> > 
> 
> > image
> 
> 
> 
> This image is cursed.  You should probably prevent tipping over if someone is buckled to the borg.  
> 
> 
Why not just have it act as if the borg used *spin? Its logical.

**Comment by SuperNovaa41** [comment]
wtf i love JohnFulpWillard

**Comment by AdamElTablawy** [comment]
this is really good actually because it will end 'borgs are overpowered' debates forever and screaming 'HELP' as a borg that's stuck upside down actually sounds really funny

**Comment by Cheshify** [comment]
Can they still use modules when flipped? Bolt and open doors? Salty borgs might just bolt you in until you reflip them again.

Second question, can we please flip the AI, not that it will do anything it'll just be funny.

**Comment by TheDavusi** [comment]
G

**Comment by squi1-rasb** [comment]
:(

**Comment by remove32** [comment]
spite thumbs down cope

**Comment by Plejek** [comment]
good pull request

**Comment by Donpedrito** [comment]
borg main here, this is funny do it

**Comment by JosephJomama** [comment]
> Can they still use modules when flipped? Bolt and open doors? Salty borgs might just bolt you in until you reflip them again.

Law 2 open door

**Comment by theOOZ** [comment]
would benefit a lot from a define like the wrecked state that imply the model has a tipped icon state, (regardless of if they actually transform 180 or not.) it's not very necessary but would be nice for flavor, so we can add unique icons like flashing its lights red or something.
also in my own PR i had added this sfx when a player started tipping over `'sound/machines/buzz-sigh.ogg'`, it's fun.

**Comment by JohnFulpWillard** [comment]
> would benefit a lot from a define like the wrecked state that imply the model has a tipped icon state, (regardless of if they actually transform 180 or not.) it's not very necessary but would be nice for flavor, so we can add unique icons like flashing its lights red or something.

I suggested downstream to make a skyrat subtype component that you can edit to do whatever you want with, I dunno much about how Silicon icon state works (especially with the light overlay thing they have for eyes) so I don't really want to touch it much.

> also in my own PR i had added this sfx when a player started tipping over `'sound/machines/buzz-sigh.ogg'`, it's fun.

I saw it when you linked it earlier, I hadn't added it because Borgs can play their own SFX sounds with emotes, and didn't want to trample over that,

> lastly i had an issue where right-clicking in combat mode to tip the borg made you punch it as well, i hope that's fixed here.

They still do, I didn't think it was much of a problem since you can't harm borgs by punching.

**Comment by theOOZ** [comment]
I saw the overwrite you posted in the SR discord and I guess we could use that! i just like defines and the way skin_features are organized.
About the punching thing, it's because attack_hand in robot_defense.dm isn't passed modifier which has the right click flag and treats it the same as a left click. it would be nice if you could solve that so the right click function is _only_ tipping

**Comment by theOOZ** [comment]
> About the punching thing, it's because attack_hand in silicon_defense.dm isn't passing modifier which has the right click flag and treats it the same as a left click. it would be nice if you could solve that so the right click function is _only_ tipping

in silicon_defense.dm i'd recommend fixing it like this, but that's my brief review.
pic1

**Comment by Mothblocks** [comment]
attack_hand_secondary please

**Comment by Dorsisdwarf** [comment]
Completely unbalanced, will encourage griefing, but is EXTREMELY FUNNY as an AI main i endorse my borgs being tpped

File: code/modules/mob/living/silicon/silicon_defense.dm [hunk 3]
```diff
@@ -68,6 +68,9 @@
 		user_unbuckle_mob(buckled_mobs[1], user)
 	else
 		if(user.combat_mode)
+			//Return here as the tippable component will do the work instead.
+			if(LAZYACCESS(modifiers, RIGHT_CLICK))
```
Comment (Mothblocks) [inline code/modules/mob/living/silicon/silicon_defense.dm:72]: use `attack_hand_secondary` and change `tippable` to use a signal that `attack_hand_secondary` fires PLEASE

Comment (JohnFulpWillard) [reply code/modules/mob/living/silicon/silicon_defense.dm:72]: OK MAN OK

**Comment by TheSilverNuke** [comment]
Is it faster for crew to untip the borg than for it to untip itself? Also I'm glad that there's another way to apprehend borgs other than locking down from a robotics console. In general borg stuns and ways to detain borgs need reworked. At the moment you can either hard stun a borg or not stun it at all, which is something I hope to see changed.

**Comment by JohnFulpWillard** [comment]
> Is it faster for crew to untip the borg than for it to untip itself? Also I'm glad that there's another way to apprehend borgs other than locking down from a robotics console. In general borg stuns and ways to detain borgs need reworked. At the moment you can either hard stun a borg or not stun it at all, which is something I hope to see changed.

3 seconds to tip a borg, 2 seconds to untip. If no one untips, takes 60 seconds to untip itself. Basically the same as handcuffs.

**Comment by TheBonded** [comment]
I think the time should be a lot less. handcuffs have the requirement of actually getting the handcuffs, which most crew don't have access to. This just requires the crew to have hands. Also, with the existence of hardstuns for borgs (flashes and laser pointers), it's way too easy to stun a borg in one click and cuff em, just like the old stun baton. maybe make the timer something like 10-20 seconds?

**Comment by thgvr** [comment]
Bad faith borg players should be ahelped instead of angry grudge prs

**Comment by DrDiasyl** [comment]
Finally, borgs supremacy will end.

**Comment by TheSilverNuke** [comment]
> Also, with the existence of hardstuns for borgs (flashes and laser pointers), it's way too easy to stun a borg in one click and cuff em, just like the old stun baton.

See this is probably my biggest problem with the PR. Borgs have no soft stun. With handcuffs if you only soft stun a player they can still escape unless you go in again for another stun. This is effectively giving all players that "win-stick" that they complain about with emagged engi-borgs.

File: code/datums/components/tippable.dm [hunk 4]
```diff
@@ -36,10 +37,10 @@
 	src.post_untipped_callback = post_untipped_callback
 
 /datum/component/tippable/RegisterWithParent()
-	RegisterSignal(parent, COMSIG_ATOM_ATTACK_HAND, .proc/interact_with_tippable)
+	RegisterSignal(parent, COMSIG_ATOM_ATTACK_HAND_SECONDARY, .proc/interact_with_tippable)
 
 /datum/component/tippable/UnregisterFromParent()
-	UnregisterSignal(parent, COMSIG_ATOM_ATTACK_HAND)
+	UnregisterSignal(parent, COMSIG_ATOM_ATTACK_HAND_SECONDARY)
```
Comment (MrMelbert) [inline code/datums/components/tippable.dm:43]: you're gonna need to change `interact_with_tippable` as well, iirc it was set up to tip on right click right on left click

might as well just change it to tip on right click, untip on right click if they're currently tipped

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by SgtHunk** [comment]
> This is effectively giving all players that \"win-stick\" that they complain about with emagged engi-borgs.

if a borg stays still for 3 seconds and lets itself get tipped then that sounds like a skill issue tbh
besides tipping over sounds like a lot friendlier of an alternative to straight up dismantling or blowing the Borg up (the latter which you can do in two clicks of a borg console)

**Comment by TheSilverNuke** [comment]
> if a borg stays still for 3 seconds and lets itself get tipped then that sounds like a skill issue tbh

I don't know if you knew this but a laser pointer, flash, or EMP will stun a borg for longer than three seconds

**Comment by Shadowflame909** [comment]
This functionality should be added to a borg tip over stick, that's craft able with metal and breaks after one use. Boom now borgs have handcuffs

**Comment by nianjiilical** [comment]
> This functionality should be added to a borg tip over stick, that's craft able with metal and breaks after one use. Boom now borgs have handcuffs

silicon spatula

**Comment by itseasytosee** [comment]
> > Can they still use modules when flipped? Bolt and open doors? Salty borgs might just bolt you in until you reflip them again.
> 
> Law 2 open door

law 2, stay still for 2.5 seconds

**Comment by necromanceranne** [comment]
Even less justification for sec to kill unruly borgs. Literally able to detain them without hardstunning. What a great change.

**Comment by zombiegums** [comment]
Anyone who seethes about borgs being OP if/when this gets merged needs to git gud. 

60 seconds is too long to tip oneself upright, however. I'd be miffed if some bellend tipped me whilst I was in the middle of surgery as a mediborg (mandating I stay still), and had to wait that long before I could continue with what I was doing.

**Comment by JohnFulpWillard** [comment]
> 60 seconds is too long to tip oneself upright, however. I'd be miffed if some bellend tipped me whilst I was in the middle of surgery as a mediborg (mandating I stay still), and had to wait that long before I could continue with what I was doing.

Lucky for you, being immobilized doesn't prevent you from doing surgery, and the person you're doing surgery on, assuming still awake, can untip you themselves.

**Comment by CameronWoof** [comment]
> Even less justification for sec to kill unruly borgs. Literally able to detain them without hardstunning repeatedly. What a great change.

I like the idea of silicon handcuffs/some kind of restraining bolt that provides this function, but I don't like it being an easy grief tool in the hands of all crew.

That you can order a borg to stay still long enough to tip them means the use case for this is almost purely to grief law-abiding silicon players rather than to legitimately disable unruly ones.

**Comment by Kubisopplay** [comment]
Absolutely grief PR, that will make rare good silicons even rarer when they get griefed for no reason. Also handcuffs without any item that can be repeated easily, i can see shitters flash and tip borgs just to "make the silicon suffer". You won't make silicon players better by making it harder to keep calm, while the shitter ones will just grief more to have fun

**Comment by Nexofas** [comment]
self_right_time = 60 SECONDS)
This opens a lot of room for griefing. No.

**Comment by JohnFulpWillard** [comment]
> self_right_time = 60 SECONDS) This opens a lot of room for griefing. No.

I've already discussed this with many different people everywhere, I'm not closing this PR because a non-contributor has told me "no"

**Comment by Nexofas** [comment]
> > self_right_time = 60 SECONDS) This opens a lot of room for griefing. No.
> 
> I've already discussed this with many different people everywhere, I'm not closing this PR because a non-contributor has told me "no"

Yes I am not a contributor. Noone needs to fix bugs in order to see if stunning a borg for 60 seconds without any tools for the price of 3 seconds is bad for the game or not.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 5]
```diff
@@ -4,6 +4,11 @@
 	spark_system.attach(src)
 
 	ADD_TRAIT(src, TRAIT_CAN_STRIP, INNATE_TRAIT)
+	AddComponent(/datum/component/tippable, \
+		tip_time = 3 SECONDS, \
+		untip_time = 2 SECONDS, \
+		self_right_time = 60 SECONDS, \
```
Comment (ATH1909) [inline code/modules/mob/living/silicon/robot/robot.dm:10]: fuck off

**Comment by Kubisopplay** [comment]
Also, the bad faith implied in this PR is astounding. One player being able to do the same job as another player, with much tighter rules and restrictions is a reason to give literally anyone else a way to grief them that isn't covered by the rules.

**Comment by JohnFulpWillard** [comment]
> Also, the bad faith implied in this PR is astounding. One player being able to do the same job as another player, with much tighter rules and restrictions is a reason to give literally anyone else a way to grief them that isn't covered by the rules.

last i checked, salt shakers don't talk

**Comment by realthomas69** [comment]
> Yes I am not a contributor. Noone needs to fix bugs in order to see if stunning a borg for 60 seconds without any tools for the price of 3 seconds is bad for the game or not.

why is it bad exactly? theres no way they will be able to tip you over if you simply move out of the way, if they have a flash, then they could easily kill you anyway, so it makes little difference

**Comment by TheBonded** [comment]
> > Yes I am not a contributor. Noone needs to fix bugs in order to see if stunning a borg for 60 seconds without any tools for the price of 3 seconds is bad for the game or not.
> 
> why is it bad exactly? theres no way they will be able to tip you over if you simply move out of the way, if they have a flash, then they could easily kill you anyway, so it makes little difference

because if you are on asimov and they tell you to stand still for 3 seconds you have to follow law 2.

**Comment by SgtHunk** [comment]
> because if you are on asimov and they tell you to stand still for 3 seconds you have to follow law 2.

Similarly, if they tell you to stand still for 60 seconds you have to follow Law 2. Your point?

**Comment by SuperNovaa41** [comment]
The borg mains salting in this pr just proves that this should be merged

**Comment by CameronWoof** [comment]
> > because if you are on asimov and they tell you to stand still for 3 seconds you have to follow law 2.
> 
> Similarly, if they tell you to stand still for 60 seconds you have to follow Law 2. Your point?

You could argue, I think successfully, that such an order would run afoul of silicon protections ("Obviously unreasonable or obnoxious orders (collect all X, do Y meaningless task).") whereas where something like ordering a cyborg to allow themselves to be tipped over might not. In either case, the comic demonstrating the mindset behind this PR seems to indicate that this will be used to grief law-abiding silicons who the user deems is annoying to them (for which no objective measure exists).

**Comment by JohnFulpWillard** [comment]
> this will be used to grief law-abiding silicons who the user deems is annoying to them (for which no objective measure exists).

isn't that the point of admins? same reason why you wouldn't be law 2'ing a borg to stand in place for 60 seconds?

**Comment by CameronWoof** [comment]
> > this will be used to grief law-abiding silicons who the user deems is annoying to them (for which no objective measure exists).
> 
> isn't that the point of admins? same reason why you wouldn't be law 2'ing a borg to stand in place for 60 seconds?
> 
> Not to mention that other players can just like... unflip them?

If the PR is to be believed, it's the express specific purpose of the feature, which is what makes me apprehensive.

**Comment by Nexofas** [comment]
> > Yes I am not a contributor. Noone needs to fix bugs in order to see if stunning a borg for 60 seconds without any tools for the price of 3 seconds is bad for the game or not.
> 
> why is it bad exactly? theres no way they will be able to tip you over if you simply move out of the way, if they have a flash, then they could easily kill you anyway, so it makes little difference

because this mechanic stands as a grief/fun mechanic rather than a fight mechanic versus borgs. If you want to comprehend a borg you can lock it, flash it, emp it. There are no "legcuff" mechanic for borgs yets. A mechanic that would "legcuff" them would be healthy for the game, but that still would need a tool like a bola to make the fight reward the one who was more prepared. Giving humanoid beings just shoving for 3 seconds to stun borgs for 60 seconds is in bad faith. This flip duratio at top should be 5 seconds.

File: code/datums/components/tippable.dm [hunk 6]
```diff
@@ -68,7 +68,7 @@
 
 	if(is_tipped)
 		INVOKE_ASYNC(src, .proc/try_untip, source, user)
-	else if(LAZYACCESS(modifiers, RIGHT_CLICK))
+	else
 		INVOKE_ASYNC(src, .proc/try_tip, source, user)
 
 	return COMPONENT_CANCEL_ATTACK_CHAIN
```
Comment (Mothblocks) [inline code/datums/components/tippable.dm:74]: @JohnFulpWillard This needs to get changed to `return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN`, I'm not sure what this value is set to but it's the wrong enum

Comment (JohnFulpWillard) [reply code/datums/components/tippable.dm:74]: I THOUGHT I DID IT WTF

Comment (LemonInTheDark) [reply code/datums/components/tippable.dm:74]: lmao, lol said the scorpion

**Comment by bluetshirtguy** [comment]
funny pr but the comics make you look like a baby

**Comment by optimumtact** [comment]
> funny pr but the comics make you look like a baby

lmao, lol said the scorpion

**Comment by Summoners** [comment]
This is truly one of the dumbest things to ever be pushed into this game and I've seen some dumb things over the many years. The idea is fine on paper but half the playerbase, especially on servers like Sibyl, have no remorse nor care for their fellow players. I have seen the greytiding shitters already bully random borgs by just doing "law 2 stand there" and flipping them everytime they have a chance. I play AI as my main and it actually is annoying to see other players being just fucked over for 40 minutes because some random assistant or even security officer decides its funny to flip Asimov borgs.

**Comment by scriptis** [comment]
this has literally done nothing but encourage griefing players

i can think of literally no situation where this makes the game better for anyone other than people that need a rule 1 ban in the first place

**Comment by itseasytosee** [comment]
> this has literally done nothing but encourage griefing players
> 
> i can think of literally no situation where this makes the game better for anyone other than people that need a rule 1 ban in the first place

Did you not read the comics?

**Comment by TheBonded** [comment]
@scriptis maybe this'll give admins more reason to ban those people who already need a rule 1 ban, which just means it'll help weed out people we don't want.

**Comment by CameronWoof** [comment]
> @scriptis maybe this'll give admins more reason to ban those people who already need a rule 1 ban, which just means it'll help weed out people we don't want.

And tell them what, exactly? "Please don't use this feature in the way it was explicitly designed and intended to be used"?

**Comment by Dorsisdwarf** [comment]
It's still funny, im not sorry borgs

**Comment by scriptis** [comment]
> > ...
> 
> Did you not read the comics?

Janiborg: nobody plays janiborg and janiborgs don't talk.
Mediborg: mediborgs don't talk.
Engiborg: You are an engineer. Build an exofab and then flash the borg.
Roboticist: You are a roboticist. Flash the borg.

The reality imposed by this PR:

You are a janiborg. Slaughter demons are destroying the station. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
You are a mediborg. You are trying to save the captain from dying in maintenance. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
You are an engiborg. The engine has been sabotaged. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
You are a default cyborg. Get a job, idiot. The crew dies anyway. Everyone is frustrated with this.
Somebody blows the borgs anyway, because somebody said AI MALF thirty seconds into the shift.
Somebody flashes you from behind and emags you anyway, because directional flashes don't apply to borgs.
The crew dies.
Everyone is frustrated with this.

**Comment by TheBonded** [comment]
> > @scriptis maybe this'll give admins more reason to ban those people who already need a rule 1 ban, which just means it'll help weed out people we don't want.
> 
> And tell them what, exactly? "Please don't use this feature in the way it was explicitly designed and intended to be used"?

@CameronWoof  if they already need a rule 1 ban, they'll likely get caught doing things like tipping mediborgs who are dragging bodies away from a blob, or tipping the engiborg who is trying to stop the SM from delamming. So the admin'll tell em to stop, maybe give em a few days ban for making the engiborg unable to stop the tesla-singulo delam, and all will be well. Once those people are weeded out, this feature'll be used for funny things and not grief.

**Comment by Mothblocks** [comment]
> You are a janiborg. Slaughter demons are destroying the station. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
You are a mediborg. You are trying to save the captain from dying in maintenance. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
You are an engiborg. The engine has been sabotaged. A random assistant tips you over. The crew dies. Everyone is frustrated with this.

You can literally make all these arguments against shoving normal humans as well. Administration already deals with cases like these regularly.

**Comment by Bawhoppen** [comment]
Gotta have a sense of humor when dealing with SS13, development and otherwise, or else you'll go insane.

**Comment by itseasytosee** [comment]
> > > ...
> > 
> > 
> > Did you not read the comics?
> 
> Janiborg: nobody plays janiborg and janiborgs don't talk. Mediborg: mediborgs don't talk. Engiborg: You are an engineer. Build an exofab and then flash the borg. Roboticist: You are a roboticist. Flash the borg.
> 
> The reality imposed by this PR:
> 
> You are a janiborg. Slaughter demons are destroying the station. A random assistant tips you over. The crew dies. Everyone is frustrated with this. You are a mediborg. You are trying to save the captain from dying in maintenance. A random assistant tips you over. The crew dies. Everyone is frustrated with this. You are an engiborg. The engine has been sabotaged. A random assistant tips you over. The crew dies. Everyone is frustrated with this. You are a default cyborg. Get a job, idiot. The crew dies anyway. Everyone is frustrated with this. Somebody blows the borgs anyway, because somebody said AI MALF thirty seconds into the shift. Somebody flashes you from behind and emags you anyway, because directional flashes don't apply to borgs. The crew dies. Everyone is frustrated with this.

All in all, I'm against using the phrase "cope" on its own, so I'm not going to.

**Comment by Dorsisdwarf** [comment]
> > You are a janiborg. Slaughter demons are destroying the station. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
> > You are a mediborg. You are trying to save the captain from dying in maintenance. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
> > You are an engiborg. The engine has been sabotaged. A random assistant tips you over. The crew dies. Everyone is frustrated with this.
> 
> You can literally make all these arguments against shoving normal humans as well. Administration already deals with cases like these regularly.

yeah, but shoving a human lasts 1 second, not 60

**Comment by Shadowflame909** [comment]
didn't this get nerfed to half its time by borgs beeping

**Pull request merged**